changelog.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## Changes in 0.17.3
+2019-08-15
+
+ * Performance improvements to inter-node data journaling.
+ * Bug: Fix prometheus module label equality searches for values beginning
```

Comment:
What's a "label"? Should this maybe be "tag equality searches"?

Reply:
prometheus doesn't have tags. it has labels

Reply:
I see, using the prometheus terminology. Got it.